apps/tailwind-components/components/Input.vue

```diff
@@ -0,0 +1,116 @@
+<script setup lang="ts">
+import type { IInputProps, IValueLabel } from "~/types/types";
+import type { CellValueType } from "metadata-utils/src/types";
+const modelValue = defineModel<any>();
+const props = withDefaults(
+  defineProps<
+    IInputProps & {
+      type: CellValueType;
+      label?: string;
+      description?: string | null;
+      describedBy?: string;
```

Comment:
Double check if still necessary

Reply:
Not sure about label and description, but ARIA props should be required or handled in this component. If we remove them (i.e., no long a prop), then these elements will never be semantically linked. It would be nice to have a mechanism that handles these attributes separately so that they aren't props and we don't have to add them manually everywhere.